assets/rpip-10/expense_2.md

```diff
@@ -0,0 +1,18 @@
+This expense has two parts:
+- Transfer the sum of periods 12, 13, 14, 15 in the `IMC amount to make up` column below to the IMC
+- Transfer the sum of periods 12, 13, 14, 15 in the `GMC amount to make up` column below to the GMC
+
+## Rationale
+There was a mismatch between the actual budget (in RPIP) and the intent (in vote text) as explained
+in <https://dao.rocketpool.net/t/intended-vs-actual-budget-mismatch-in-rpip-10/2053?u=valdorff>.
+
+This expense compensates the difference during periods affected before the budget was updated.
+
+| Period | RPL to treasury | IMC amount to make up | GMC amount to make up |
+|--------|-----------------|-----------------------|-----------------------|
+| n      | x               | `x*(.34-.27)=.07x`    | `x*(.205-.16)=.045x`  | 
```

Comment:
Where do these formulas come from? I can't see them easily in the linked explanation.

Reply:
Added a small bit of explanation.